test-data/unit/pythoneval.test

```diff
@@ -1251,7 +1251,7 @@ bar: Type[Union[A, B]]
 res = bar * 4
 reveal_type(res)
 [out]
-_testMetaclassOpAccessUnion.py:16: note: Revealed type is "builtins.str | builtins.int"
+_testMetaclassOpAccessUnion.py:16: note: Revealed type is "Union[builtins.str, builtins.int]"
```

Comment:
why has this changed back to old union? isnt that the opposite of what this change is supposed to do?

Reply:
https://github.com/KotlinIsland/basedmypy/pull/290#discussion_r863475456